src/DatadogLoggingService.js

```diff
@@ -27,6 +27,13 @@ class DatadogLoggingService extends NewRelicLoggingService {
     this.initialize();
   }
 
+  // to read more about the use cases for beforeSend
```

Comment:
[nit/suggestion] Think it's worth including an example use case referenced in the [related issue](https://github.com/edx/edx-arch-experiments/issues/698#issuecomment-2187124955) around ignoring certain messages matching a configured regex.

> [aside] fwiw, slightly related, we might also want to refactor to rely on beforeSend in the common/base DatadogLoggingService class to implement the `IGNORED_ERROR_REGEX` stuff to ignore certain error messages but that's slightly more tangential to this specific request 😄

```suggestion
  // to read more about the use cases for beforeSend (e.g., discarding frontend errors matching the optional `IGNORED_ERROR_REGEX` configuration, currently implemented in `logError` below).
```